src/main/kotlin/no/nav/tilleggsstonader/sak/utbetaling/iverksetting/HentStatusFraIverksettingTask.kt

```diff
@@ -0,0 +1,54 @@
+package no.nav.tilleggsstonader.sak.utbetaling.iverksetting
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import no.nav.familie.prosessering.AsyncTaskStep
+import no.nav.familie.prosessering.TaskStepBeskrivelse
+import no.nav.familie.prosessering.domene.Task
+import no.nav.tilleggsstonader.kontrakter.felles.ObjectMapperProvider.objectMapper
+import org.springframework.stereotype.Service
+import java.time.LocalDateTime
+import java.util.Properties
+import java.util.UUID
+
+@Service
+@TaskStepBeskrivelse(
+    taskStepType = HentStatusFraIverksettingTask.TYPE,
+    maxAntallFeil = 50,
+    settTilManuellOppfølgning = true,
+    triggerTidVedFeilISekunder = 31L,
```

Comment:
Hvor raskt kan man forvente at iverksetting blir ok? Nå som vi ikke lengre venter på denne for å ferdigstille behandlingen kan vi kanskje vurdere å vente litt lengre tid?

Reply:
Setter denne til 60. Tror det ska gå relativt raskt